tests/test_multiple_learn.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+
+from stable_baselines import A2C, ACER, ACKTR, PPO2
+from stable_baselines.common.identity_env import IdentityEnv, IdentityEnvBox
+from stable_baselines.common.vec_env import DummyVecEnv
+
+# TODO: Fix multiple-learn on commented-out models (Issue #619).
+MODEL_LIST = [
+    A2C,
+    ACER,
+    ACKTR,
+    PPO2,
+
+    # MPI-based models, which use traj_segment_generator instead of Runner.
+    #
+    # PPO1,
+    # TRPO,
+
+    # Off-policy models, which don't use Runner but reset every .learn() anyways.
+    #
+    # DDPG,
+    # SAC,
+    # TD3,
+]
+
+
+@pytest.mark.parametrize("model_class", MODEL_LIST)
+def test_model_multiple_learn_no_reset(model_class):
+    """Check that when we call learn multiple times, we don't unnecessarily
+    reset the environment.
+    """
+    if model_class is ACER:
+        def make_env():
+            return IdentityEnv(ep_length=1e10, dim=2)
+    else:
+        def make_env():
+            return IdentityEnvBox(ep_length=1e10)
+    venv = DummyVecEnv([make_env])
```

Comment:
Might be simpler to create `env` and then embed it in `venv` given we need to retrieve it anyway? e.g:

```
env = make_env()
venv = DummyVecEnv([lambda: env])
```